### DISCUSSION TITLE
Silencing the tunes the 5M plays

### DISCUSSION THREAD

**Discussion by electricblue71** [xblax/flashforge_ad5m_klipper_mod#251] [Development]
I really like the Klipper Mod so far on my 5M. However the song that plays when you first start up the printer and when a print job is done is annoying and loud. Sometimes I print early in the morning or late at night and I don't want to wake up the whole house.

Is there a way to turn that off? I was able to in the OEM firmware, so miss having the ability to do that in the Klipper Mod.

**Comment by vennen894** [comment]
Change your end print macro from END_PRINT to _COMMON_END_PRINT

**Reply by electricblue71** [reply]
Thanks, will give that a try.
